bumble/pandora/l2cap.py

```diff
@@ -61,6 +61,12 @@ def __init__(self, device: Device, config: Config) -> None:
         self.config = config
         self.sdu_queue: asyncio.Queue = asyncio.Queue()
 
+    def on_channel_sdu(self, sdu):
+        async def handle_sdu():
+            await self.sdu_queue.put(sdu)
+
+        asyncio.create_task(handle_sdu())
```

Comment:
Please use synchronized Queue instead:
https://docs.python.org/3/library/queue.html

```suggestion
    def on_channel_sdu(self, sdu):
        self.sdu_queue.put_nowait(sdu)
```